types/params.go

```diff
@@ -7,40 +7,68 @@ import (
 	cmtypes "github.com/cometbft/cometbft/types"
 )
 
+var (
+	// ErrMaxBytesCannotBeZero indicates that the block's MaxBytes cannot be 0.
+	ErrMaxBytesCannotBeZero = errors.New("block.MaxBytes cannot be 0")
+
+	// ErrMaxBytesInvalid indicates that the block's MaxBytes must be either -1 or greater than 0.
+	ErrMaxBytesInvalid = errors.New("block.MaxBytes must be -1 or greater than 0")
+
+	// ErrMaxBytesTooBig indicates that the block's MaxBytes exceeds the maximum allowed size.
+	ErrMaxBytesTooBig = errors.New("block.MaxBytes is too big")
```

Comment:
is there a way that we can provide direction here? either by stating the max size or saying how to find it?

Reply:
Added a link to https://github.com/cometbft/cometbft/blob/2cd0d1a33cdb6a2c76e6e162d892624492c26290/types/params.go#L16 in the explaining comment.


Side note:

This is originally from here: https://github.com/cometbft/cometbft/blob/830c650e76a3049a9e1bb7b5e84416691bf6c187/types/params.go#L155

Interestingly, they only check for error vs no error case: https://github.com/cometbft/cometbft/blob/830c650e76a3049a9e1bb7b5e84416691bf6c187/types/params_test.go#L48